web/app/templates/checkpoints.html

```diff
@@ -14,10 +14,10 @@ <h3>Model Checkpoints</h3>
     <table class="table table-hover" style="width:auto">
         {% for checkpoint in checkpoints %}
             <tr>
-                <th style="vertical-align: middle;">{{checkpoint}}</th>
+                <th style="vertical-align: middle;">{{checkpoint[1]}}</th>
```

Comment:
Can you modify the functions that return a row from the table so that they instead return a dictionary or some other Python object that would be easier to interpret?